src/lib/server-impl.ts

```diff
@@ -36,6 +37,13 @@ async function createApp(
     const stores = createStores(config, db);
     const services = createServices(stores, config);
 
+    const cdnPrefix = config.server.cdnPrefix;
+    let indexHTMLOverride: string;
+    if (cdnPrefix) {
+        const res = await fetch(`${cdnPrefix}/index.html`);
+        indexHTMLOverride = await res.text();
+    }
```

Comment:
this is not the right place for this loading. Will move it to a separate tool for loading the indexHTML file.